[no-issue] feat: add SIEM Azion en (#548)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/observe/siem/siem.mdx

---
title: SIEM Azion
description: Analyze data from dashboards with raw and aggregated data.

> meta_tags: siem, edge computing, observe, observability, metrics, data, events

Reviewer: ```suggestion
meta_tags: siem, observe, observability, metrics, data, events
```

namespace: docs_products_siem_azion
permalink: /documentation/products/siem-azion/
---

> **Azion SIEM** allows the visualization of aggregated and raw data through data from [Real-Time Metrics](/en/documentation/products/real-time-metrics/) and [Real-Time Events](/en/documentation/products/real-time-events/).

Reviewer: ```suggestion
**Azion SIEM** allows you to expand the visualization of accesses made to your edge applications. By correlating data available on [Real-Time Metrics](/en/documentation/products/real-time-metrics/) and [Real-Time Events](/en/documentation/products/real-time-events/), you can amplify your analysis of security, access volume, health, degradation, and prediction.
```

> The **Azion SIEM** allows the visualization of aggregated and raw data through data from [Real-Time Metrics](/en/documentation/products/real-time-metrics/) and [Real-Time Events](/en/documentation/products/real-time-events/).

Reviewer: ```suggestion
**Azion SIEM** allows the visualization of aggregated and raw data through data from [Real-Time Metrics](/en/documentation/products/real-time-metrics/) and [Real-Time Events](/en/documentation/products/real-time-events/).
```

## Dashboards

> Each dashboard has a set of graphs that allows for a more in-depth analysis of specific metrics and metric categories. Below are the available dashboards.

Reviewer 2: ```suggestion
Each dashboard has a set of graphs that allows for a more in-depth analysis of specific metrics and metric categories. The available dashboards are detailed below.
```

> Each dashboard has a set of graphs that allows for a more in-depth analysis of specific metrics and metric categories. The available dashboards are detailed below.

Reviewer: ```suggestion
Each dashboard has a set of graphs and tables that allows you to monitor your applications according to different observability needs. The available dashboards are detailed below.
```

- Requests by Stacked Status < - Upstream status
- Requests (Total x Blocked)
- Requests per Second (Total x Blocked)
- Blocked Requests Summary
    - Blocked Requests by IP
    - Blocked Requests by URI
- Top 20 Breakdown
    - Top 20 ASN
    - Top 20 User Agents
    - Top 20 IP
    - Top 20 URI Most Accessed
    - Top 20 Remote Port
    - Top 20 Requests per IP Address
- Requests by Country
    - Requests by Country (table)
    - Requests by Country (Upstream response, table)
    - Requests by Country (graph)
    - Requests by Country (Upstream response, graph)
- Status Breakdown
    - 2XX
    - Top 20 Requests by URI (Status 2xx)
    - 3XX
    - Top 20 Requests by URI (Status 3xx)
    - 4XX
    - Top 20 Requests by URI (Status 4xx)
    - 5XX
    - Top 20 Requests by URI (Status 5xx)
- Log View

### Follow-up by Error Status

Presents data that assists in the analysis of requests when observing status responses in one or more edge applications.

The following graphs are available:

- Requests/sec by Top 20 ASN
- Requests by Top 20 ASN
- Total Requests by Top 100 ASN
- Total Requests by ASN %
- Total Requests by Top 100 Remote Address

#### Follow-up by Error Status - Drill Down

> Complements the Follow-up by Error Status dashboard with graphs with more specific pre-configured filters.

Reviewer 3: ```suggestion
Complements the **Follow-up by Error Status** dashboard with graphs with more specific pre-configured filters.
```

The following graphs are available:

- Requests by Status
- Requests by Status %
- Request Info (Last 100)
- Top 20
    - Remote Address
    - Status
    - Country
    - Remote Port
    - User Agent
    - Request URI

### Telemetry and Service Degradation

Presents data on the averages of key request information and status information, and top X status x URIs.

The following graphs are available:

- AVG Request Time
- Top 20 Summary
    - Requests by Country - AVG
    - Region of Brazil - AVG
    - Requests by Remote Address - AVG
    - URIs by Count - AVG
- Status breakdown
    - 2XX
    - Top 100 Requests By URI no QS (Status 2xx)
    - 3XX
    - Top 100 Requests by URI no QS (Status 3xx)
    - 4XX
    - Top 100 Requests by URI no QS (Status 4xx)
    - 5XX
    - Top 100 Requests by URI no QS (Status 5xx)
- Log View

> ### Login - Monitoring
> 
> Displays data related to requests during the login process.
> 
> The following graphs are available:
> 
> - Hits by PATH
> - Hits by Status
> - Hits by Login Attempt
> - Hits by Success login
> - Hits by Failed login
> - Hits by Success
> - Hits by Failed
> - Average Hits per second by Client ID+PATH (Not Blocked)
> - Average Hits per second by Client-ID (Not Blocked)
> - AVG Hits/sec by IP + Client ID (Not Blocked)
> - Top 20 Average Hits Client ID+UA (Not Blocked)
> - Hits per Session Code by Client ID (Not Blocked)
> - Hits per repeated Session Code by Client ID (Not Blocked)
> - Top 20 Requests per User-Agent
> - Top 20 Requests per IP Address
> - Average Hits per second by CLIENT-ID x STATE (Not Blocked)
> - Average Hits per second by CLIENT-ID x NONCE (Not Blocked)
> 

Reviewer: ```suggestion
```

### Edge Survey

> Presents data related to the behavior of the Azion edge in processing requests. This dashboard is directly related to the use of **Origin Shield** on the account.

Reviewer 3: maybe add link to Origin Shield? [align with PT]

The following graphs are available:

- Requests by Timeout Errors in Azion - Proxy Status
- Connect time average of error requests
- Requests by Timeout in Azion - Proxy Status

## Alerts

> Specific alerts can be configured. The creation of alerts covers:

Reviewer: ```suggestion
Specific alerts can be configured and are received via email. The creation of alerts covers:
```
